jwql/utils/constants.py

```diff
@@ -238,6 +238,9 @@
                                       'G235H', 'G395H', 'PRISM']
                           }
 
+# Filename extensions for guider data
+GUIDER_FILENAME_TYPE = ['guider', 'gs-fg', 'gs-track', 'gs-id', 'gs-acq1', 'gs-acq2']
```

Comment:
'guider' will be in science files, while the 'gs-' options will be in guide star data. So it seems like you want to take out the 'guider' option given the way you look for matches in forms.py, right?

Reply:
Thanks for information @bhilbert4, fixed this in [f0c68d8](https://github.com/spacetelescope/jwql/pull/1010/commits/f0c68d8efe2f6f431b1363af00073bfb89a2bcd0)